usr/sbin/p11sak/p11sak.c

```diff
@@ -1338,7 +1356,7 @@ static const struct p11sak_opt p11sak_list_key_opts[] = {
       .arg =  { .type = ARG_TYPE_PLAIN, .required = false,
                 .value.plain = &opt_long, },
       .description = "List keys in long (detailed) format.", },
-    { .short_opt = 0, .long_opt = "detailed-uri", .required = false,
+    { .short_opt = 'u', .long_opt = "detailed-uri", .required = false,
```

Comment:
Same here, any specific reason?

Reply:
Please update the man page as well if you change the arguments.

Reply:
Same here, if there is no technical need for it, I would tent to leave it as it was before.